README.md

```diff
@@ -1,10 +1,4 @@
-# Ressources
 
-Liste de liens, blogs, outils, ressources, ...
-
-N'hésitez pas d'ouvrir des Issues ou des Pull requests
-
----
```

Comment:
Pourquoi supprimer l'en-tête ?

Reply:
Ouai bizarre, comment on fait pour revenir en arrière la dessus ?

Reply:
Il faudrait supprimer le commit. Ça se fait avec *rebase*. Si quelqu'un veut le faire, qu'il le fasse, sinon, je le ferai lundi. (Mon WE est chargé)